File: Algorithms/Searching/Linear_Search/LinearSearch.java
public class LinearSearch{

public static int linearsearch(int arr[],int ele){

for(int i=0;i<arr.length;i++)
{
if(arr[i]==ele){
return i;
}
}
return -1;
}
public static void main(String args[]){
int[] arr = new int[]{ 1,2,3,4,5,6,7,8,9,10 };
System.out.print("Poistion of element is:") ;
System.out.print(linearsearch(arr,5));

}




}

File: Algorithms/Searching/Linear_Search/Linear_Search.js
/*JavaScript Implementation of linear Search */

function linearSearch(arr, item) {
  // Go through all the elements of arr to look for item.
  for (var i = 0; i < arr.length; i++) {
    if (arr[i] === item) { // Found it!
      console.log( "Element" + item + " found at index " + i) ;
      return ;
    }
  }
  
  // Item not found in the array.
  console.log("Element" + item + " is not present in the given array");
}

function main(){
    var arr = [5,10,15,20,25];
    var item = 20;
    linearSearch(arr,item);
}

main();